.drone.star

```diff
@@ -224,6 +236,61 @@ def testing(ctx, module):
     },
   }
 
+def uploadCoverage(ctx):
+  return {
+    'kind': 'pipeline',
+    'type': 'docker',
+    'name': 'upload-coverage',
+    'platform': {
+      'os': 'linux',
+      'arch': 'amd64',
+    },
+    'steps': [
+      {
+        'name': 'sync-from-cache',
+        'image': 'minio/mc',
+        'environment': {
+          'MC_HOST_cache': {
+            'from_secret': 'cache_s3_connection_url'
+          }
+        },
+        'commands': [
+          'mc mirror cache/cache/%s/%s/coverage coverage/' % (ctx.repo.slug, ctx.build.commit + '-${DRONE_BUILD_NUMBER}'),
```

Comment:
```suggestion
          'mkdir -p coverage'
          'mc mirror cache/cache/%s/%s/coverage coverage/' % (ctx.repo.slug, ctx.build.commit + '-${DRONE_BUILD_NUMBER}'),
```

Maybe make the local `coverage` dir first, to avoid the annoying error output from `mc mirror`